src/queries/user/joinedCircles.ts

```diff
@@ -0,0 +1,29 @@
+import { PRICE_STATE } from 'common/enums'
+import { UserToJoinedCirclesResolver } from 'definitions'
+
+const resolver: UserToJoinedCirclesResolver = async (
+  { id },
+  _,
+  { dataSources: { atomService }, knex }
+) => {
+  if (!id) {
+    return []
+  }
+
+  const circleIds = await knex
+    .select('price.circle_id')
+    .from('circle_subscription_item as csi')
+    .innerJoin('price', 'pirce.id', 'csi.price_id')
```

Comment:
only items attached to the active subscription?

Reply:
Added.